spec/order_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'order'
+
+RSpec.describe Order do
+  let(:order) { described_class.new(menu) }
+  let(:menu) { double :menu, :dishes => dishes_on_menu }
```

Comment:
alternate syntax: @menu = double("Menu", dishes: dishes_on_menu)

I like that you have the :dishes return just a simple phrase rather than fake dishes. You are only testing what you need to test to get the job done :)